indico/modules/attachments/controllers/management/event.py

```diff
@@ -97,3 +101,13 @@ class RHPackageEventAttachmentsManagement(AttachmentPackageMixin, RHManageEventB
     wp = WPPackageEventAttachmentsManagement
     management = True
     ALLOW_LOCKED = True
+
+
+class RHSetUploadPermissions(RHManageEventBase):
+    def _process(self):
+        form = SetUploadPermissionsForm(managers_only=attachments_settings.get(self.event, 'managers_only'))
+        if form.validate_on_submit():
+            attachments_settings.set(self.event, 'managers_only', form.managers_only.data)
```

Comment:
let's use the get_all/set_multi version here. i don't want to have to update this once another setting gets added there ;)